larray/inout/excel.py

```diff
@@ -61,7 +61,7 @@ def read_excel(filepath, sheet=0, nb_axes=None, index_col=None, fill_value=np.na
         Engine to use to read the Excel file. If None (default), it will use 'xlwings' by default if the module is
         installed and relies on Pandas default reader otherwise.
     range : str, optional
-        Range in which array is stored. Used only if engine is 'xlwings'. Defaults to None.
+        Range in which array is stored. Used only if engine is 'xlwings'. Defaults to slice(None).
```

Comment:
should explain what that slice(None) means. Something like "(which means the entire sheet)"